bin/node/cli/Cargo.toml

```diff
@@ -96,7 +96,7 @@ sc-service-test = { version = "2.0.0", path = "../../../client/service/test" }
 futures = "0.3.1"
 tempfile = "3.1.0"
 assert_cmd = "0.12"
-libc = "0.2"
+nix = "0.17"
```

Comment:
Don't we need to make this an optional dependency that is only activated for unix?

Reply:
I asked myself the same question but I saw this in the source code:

https://docs.rs/nix/0.17.0/src/nix/lib.rs.html#6

The whole lib is in unix only unless I'm mistaking

Reply:
I will check on Windows to be sure

Reply:
Confirmed.